yarn-project/merkle-tree/src/pedersen.ts

```diff
@@ -9,22 +9,35 @@ import { Hasher } from '@aztec/types';
 
 /**
  * A helper class encapsulating Pedersen hash functionality.
+ * @deprecated Don't call pedersen directly in production code. Instead, create specific nicely-called WASM functions.
```

Comment:
Same suggestion as above.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2724/commits/4b722a5d79143fa4138c7c7fded119945b23d5f7